crates/sui-cost/data/estimate.toml

```diff
@@ -0,0 +1,44 @@
+[MergeCoin]
```

Comment:
for my own understanding: where do these value come from?

Reply:
The estimator basically does a trial run during test and creates the file. This is is essentially a cache of costs that are constant, or that we can use to approximate other costs.

Reply:
who runs the trial and updates this file and when?

Reply:
> who runs the trial and updates this file and when?

This runs as part of unit test. It will be integrated with snapshot testing logic so that we always update the file.